tests/test_github_requirements.py

```diff
@@ -115,29 +111,20 @@ def _test_require_approval_username(self, project, job):
         self.assertEqual(len(self.history), 0)
 
         # Add an approval (review) from derp
-        A.addReview('derp', 'APPROVE')
```

Comment:
This is changing from APPROVE to APPROVED, we need to treat the webhook trigger and the query results differently :(